src/driver-steelseries.c

```diff
@@ -157,8 +180,11 @@ steelseries_get_firmware_version(struct ratbag_device *device)
 
 	if (device_version == 1)
 		return 0;
+	else if (device_version == 2)
+		buf[0] = STEELSERIES_ID_FIRMWARE;
+	else if (device_version == 3)
+		buf[0] = STEELSERIES_ID_FIRMWARE_SHORT;
```

Comment:
Why short? Should this be `STEELSERIES_ID_FIRMWARE_PROTOCOL3` or something like that?